include/sound/soc.h

```diff
@@ -646,6 +646,11 @@ struct snd_soc_dai_link_component {
 	const char *dai_name;
 };
 
+struct snd_soc_dai_link_codec_ch_map {
+	unsigned int connected_cpu_id;
+	unsigned int ch_mask;
+};
+
```

Comment:
commit message: confusion between dai and links

Currently, ASoC supports dailinks with the following mappings:
1 cpu DAI to N codec dai
N cpu DAI to N codec DAI

But the mapping between N cpu Dais and M codec dais is not supported.